### ISSUE TITLE
Unexpected ouput and missing text on "update" command

### ISSUE THREAD

**Issue by Zasco** [memo33/sc4pac-tools#8]
Since upgrading to **0.4.5**, I get unexpected outputs (_←8_) and missing texts when using the "_update_" command. Here are examples:
```
***\sc4pac\releases\latest>sc4pac update
The following packages will be removed:
    bsc:mega-props-d66-vol03 1
    lbt:mega-prop-pack-vol01 3.0
    lbt:wal-mart-supercenter-v2 1.0
    walkingsnake:parking-textures-vol1 1
The following packages will be installed:
    bsc:essentials 2024b
    bsc:mega-props-mjb-vol01 3
    bsc:mega-props-sg-vol01 4
    bsc:textures-vol01 1-1
    simgoober:mobile-homes 1.03
Continue? [Yes/no]:
←8(1/5) Extracting bsc:mega-props-mjb-vol01 ??????
←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8      ←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8
←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8      ←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8(2/5) Extracting bsc:essentials 2024b
←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←      8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8(3/5) Extracting bsc:mega-props-sg-vol01 4
←8←8←8←8←8←8←8←8(4/5) Extracting bsc:tex      vol01 1-1
←8←8←8←8←8←8(5/5) Extracting simgoober:mobile       1.03
←8Moving extracted files to plugins folder
←8←8←8←8←8←8←8←8←8←8←8←8←8←8Done.
***\sc4pac\releases\latest>
**&**
←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8←8(1/1)       ting null-45:dbpf-loading 1.0.4
←8←8←8←8←8←8Moving extracted files to plug      lder.
←8←8←8←8←8←8←8←8←8←8Done.
```

- _sc4pac_ version: **0.4.5**
- OS: _Windows_ **8.1**
- Output of java -version:
> java version "1.8.0_431"
Java(TM) SE Runtime Environment (build 1.8.0_431-b10)
Java HotSpot(TM) 64-Bit Server VM (build 25.431-b10, mixed mode)
- Terminal: _Windows_ terminal
- Using unicode (chcp 65001) in command prompt does not solve the problem.
- Output of "java -jar jansi-2.4.1.jar":
> Jansi 2.4.1
library.jansi.path=
library.jansi.version=
Jansi native library loaded from C:\Users\Pascal\AppData\Local\Temp\jansi-2.4.1-
f915dce7737cfa81-jansi.dll
   which was auto-extracted from jar:file:/C:/Users/Pascal/Desktop/jansi-2.4.1.j
ar!/org/fusesource/jansi/internal/native/Windows/x86_64/jansi.dll
os.name= Windows 8.1, os.version= 6.3, os.arch= amd64
file.encoding= Cp1252
sun.stdout.encoding= cp850, sun.stderr.encoding= cp850
stdout.encoding= null, stderr.encoding= null
java.version= 1.8.0_431, java.vendor= Oracle Corporation, java.home= C:\Program
Files\Java\jre1.8.0_431
Console: java.io.Console@3fb6a447
jansi.graceful=
jansi.mode=
jansi.out.mode=
jansi.err.mode=
jansi.colors=
jansi.out.colors=
jansi.err.colors=
jansi.passthrough= false
jansi.strip= false
jansi.force= false
jansi.noreset= false
org.fusesource.jansi.Ansi.disable= false
IS_WINDOWS: true
IS_CONEMU: false
IS_CYGWIN: false
IS_MSYSTEM: false
isatty(STDOUT_FILENO): 1, System.out is a terminal
width(STDOUT_FILENO): 80
isatty(STDERR_FILENO): 1, System.err is a terminal
width(STDERR_FILENO): 80

**Comment by memo33** [comment]
Thanks for the report. I've looked through the changes of the upstream library "jansi" and found the relevant change (see https://github.com/fusesource/jansi/issues/226). Fortunately, it's possible to revert to the previous behavior, which I've implemented in [0.4.6-SNAPSHOT](https://github.com/memo33/sc4pac-tools/releases/tag/0.4.6-SNAPSHOT), so you can test it.

The fix might not work for the inbuilt macOS Term.app, as documented in that issue. If anyone encounters that problem, we can look into a platform-specific fix.

For clarification, the `←8` are escape codes to control cursor positioning that shouldn't normally be printed. After the progress bar is finished, I overwrite it with spaces to hide it. As the cursor is at an unexpected position due to the extra printed output, the spaces end up in the wrong place leading to some obscured characters.

**Comment by Zasco** [comment]
Solved in [0.4.6-SNAPSHOT](https://github.com/memo33/sc4pac-tools/releases/tag/0.4.6-SNAPSHOT).
